ben_lowenstein.cfdg

```diff
@@ -0,0 +1,30 @@
+startshape OUTER
```

Comment:
Your code looks great, its just worth noting how awesome it is that such a short, simple program can generate something so complex and beautiful.